db/util_testing.go

```diff
@@ -525,8 +511,14 @@ func SetupTestDBForDataStoreWithOptions(t testing.TB, tBucket *base.TestBucket,
 	return db, ctx
 }
 
-// getScopesOptions sets up a ScopesOptions from a TestBucket for use with non default collections to pass in DatabaseContextOptions.
-func getScopesOptions(t testing.TB, testBucket *base.TestBucket, numCollections int) ScopesOptions {
+// GetScopesOptions sets up a ScopesOptions from a TestBucket for use with non default collections to pass in DatabaseContextOptions.
```

Comment:
I think the 'non-default collections' part of this comment should now be removed?